src/libraries/System.Linq.Queryable/src/System/Linq/EnumerableQuery.cs

```diff
@@ -110,10 +110,9 @@ private IEnumerator<T> GetEnumerator()
             return _enumerable.GetEnumerator();
         }
 
-        public override string ToString()
+        public override string? ToString()
```

Comment:
I am still thinking this should return non null? Wondering what am I missing ...

Reply:
I wouldn't think so, `_enumerable` can be an arbitrary IEnumerable implementation which can be passed from a public constructor.